cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/embankment.json

```diff
@@ -49,5 +25,29 @@
     "elevation-units": "m",
     "bounding-office-id": "LRL",
     "nearest-city": "Davis"
+  },
+  "structure-type": {
+    "active": true,
+    "office-id": "CWMS",
+    "tooltip": "TOOLTIP_STRUCT",
+    "display-value": "STRUCT"
+  },
+  "upstream-side-slope": 15.0,
+  "downstream-side-slope": 90.0,
+  "structure-length": 25.0,
+  "height-max": 5.0,
+  "top-width": 20.0,
+  "units-id": "ft",
+  "downstream-prot-type": {
```

Comment:
We're good with "prot" as an abbreviation "term of art" etc.   I'm fine with it.  Making sure there is concensus.

Reply:
what the heck is "prot"?

Reply:
stands for "protection", I was following db and java code. I'll revise in a separate PR (trying to keep this to turbines only)

Reply:
and failing....

Reply:
Okay, we should definitely spell that out, but it can wait for now.